.github/workflows/spellcheck.yml

```
@@ -19,3 +19,7 @@ jobs:
         # typo_ci_license_key: ${{ secrets.TYPO_CI_LICENSE_KEY }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+# Any files/folders we should ignore?
+excluded_files:
+  - "rasa_sdk/**/*"
```

Thanks for the change. Adding an empty line at the end (https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file).

```suggestion
  - "rasa_sdk/**/*"
```

---

nice! I didn't know! thanks for sharing. In the same I found also this https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list